# UDPMessage.msg

uint16 messageid
uint16 uavid
float64 longitude
float64 latitude
float64 altitude 
int32 parity